keps/sig-node/127-user-namespaces/README.md

```diff
@@ -422,6 +422,34 @@ components that implement the interface.
 
 [kubeletVolumeHost-interface]: https://github.com/kubernetes/kubernetes/blob/36450ee422d57d53a3edaf960f86b356578fe996/pkg/volume/plugins.go#L322
 
+### Pod Security Standards (PSS) integration
+
+[Pod Security Standards](https://k8s.io/docs/concepts/security/pod-security-standards)
+define three different policies to broadly cover the whole security spectrum of
+Kubernetes, while the User Namespaces feature should integrate into them. This
+will happen only if the feature is graduated to GA, which _may_ result in
+changing the `Restricted` profile to disallow host user namespaces.
+
+With graduating the feature to beta, the Pod Security will relax in a controlled
+way for pods which enable user namespaces. This behavior can controlled by an API
+Server Feature Gate, which allows an early opt-in for end users. The overall
+burden to ensure that all nodes will honor user namespaces is on the cluster
+admin, though. The relaxation in detail means, that if user namespaces are
```

Comment:
Do we have a test to validate that Pod will not be admitted if we cannot set user namespaces?

Reply:
I assume we can test API server feature gates within serial tests, so I added a comment to integrate that as well.